jsx/Filter.js

```diff
@@ -39,7 +39,7 @@ class Filter extends Component {
   onFieldUpdate(name, value, id, type) {
     const searchParams = new URLSearchParams(location.search);
     const filter = JSON.parse(JSON.stringify(this.props.filter));
-    const exactMatch = (!(type === 'textbox' || type === 'date'));
+    const exactMatch = (!(type === 'textbox' || type === 'date' || type === 'select'));
```

Comment:
Why is this PR removing the exactMatch criteria for selects? If you select a dropdown with id `1` it shouldn't match id `11`

Reply:
Right now, each field in the data table is a string. Let's look at an example:

**X** has visits: `V1`, `V2`, `V3`
**Y** has visit: `V2`

If you select `V2` in the dropdown, you'd expect that both **X** and **Y** are shown in the filtered data table. However, that is not the case since at the moment, it is looking for an exact match. Thus, the only result you will see is a row for **Y**.

I agree with the case you've given above, but I also don't think we should be returning a subset of results. Any suggestions?